dftool

```diff
@@ -462,8 +462,16 @@ Transactions are signed with envvar 'DFTOOL_KEY`.
     token_symbol = B.Simpletoken.at(TOKEN_ADDR).symbol().upper()
     token_symbol = token_symbol.replace("MOCEAN", "OCEAN")
     rewards = csvs.loadRewardsCsv(CSV_DIR, token_symbol)
+
```

Comment:
"CHAINID" should no longer be an argument to "dftool dispense".
- in the docstring for "dfool" main: `dftool dispense CSV_DIR CHAINID [DFREWARDS_ADDR] [TOKEN_ADDR] - from rewards, dispense funds`
- in the docstring for "dftool dispense" itself, also: `dftool dispense CSV_DIR CHAINID [DFREWARDS_ADDR] [TOKEN_ADDR] - from rewards, dispense funds`

Accordingly, `do_dispense()` shouldn't be processing CHAINID at all